components/script/dom/readablestreamdefaultcontroller.rs

```diff
@@ -200,15 +238,38 @@ impl ReadableStreamDefaultController {
         strategy_hwm: f64,
         strategy_size: Rc<QueuingStrategySize>,
     ) -> DomRoot<ReadableStreamDefaultController> {
-        reflect_dom_object(
+        let rooted_default_controller = reflect_dom_object(
             Box::new(ReadableStreamDefaultController::new_inherited(
                 global,
                 underlying_source,
                 strategy_hwm,
                 strategy_size,
             )),
             global,
-        )
+        );
+
+        if let Some(underlying_source) = rooted_default_controller.underlying_source.get() {
+            let promise = underlying_source.call_start_algorithm(
+                Controller::ReadableStreamDefaultController(rooted_default_controller.clone()),
+            );
+            let fulfillment_handler = Box::new(StartAlgorithmFulfillmentHandler {
+                controller: Dom::from_ref(&*rooted_default_controller),
+            });
+            let rejection_handler = Box::new(StartAlgorithmRejectionHandler {
+                controller: Dom::from_ref(&*rooted_default_controller),
+            });
```

Comment:
There are some stuff need to do upon fulfillment and rejection.Do they go to the `todo!()` in Callback trait implementations?

Reply:
yes.